regulations/static/regulations/js/source/views/main/child-view.js

```diff
@@ -32,9 +31,6 @@ var ChildView = Backbone.View.extend({
                 }
 
                 this.route(this.options);
-
-                GAEvents.sendEvent('section:open', this.options.id);
-
```

Comment:
Oh, I see what you mean by this now. The issue wasn't that the function was getting called twice, but rather that a second explicit user interaction was also being called. So what was happening was the events being passed might be something like:

```
reference:click
section:open
```

Which is redundant since we also do page view tracking.